packages/driver/src/cy/net-stubbing/add-command.ts

```diff
@@ -19,14 +19,17 @@ import {
   validateStaticResponse,
   getBackendStaticResponse,
   hasStaticResponseKeys,
-  hasRouteMatcherKeys,
 } from './static-response-utils'
 import { registerEvents } from './events'
 import $errUtils from '../../cypress/error_utils'
 import $utils from '../../cypress/utils'
 
 const lowercaseFieldNames = (headers: { [fieldName: string]: any }) => _.mapKeys(headers, (v, k) => _.toLower(k))
 
+export function hasOnlyRouteMatcherKeys (obj: any) {
+  return !_.isEmpty(obj) && !_.isArray(obj) && _.isEmpty(_.omit(obj, _.concat(PLAIN_FIELDS, STRING_MATCHER_FIELDS, DICT_STRING_MATCHER_FIELDS)))
```

Comment:
will this be truthy if obj is a function?

Reply:
like this?

```js
cy.intercept('/url', () => {})
cy.intercept('/url', function () {})


nope, see `net_stubbing_spec` for examples of such usage